pysnc/record.py

```diff
@@ -63,6 +66,12 @@ def get_display_value(self) -> Any:
             return self._display_value
         return self._value
 
+    def get_link(self) -> Any:
```

Comment:
Do we want a `nostack` here as a parameter? Or is it just is what it is

Reply:
The reference links appear to be going right after `/api/now/table/{table}/{sys_id}` in the path w/o any query parameters.